src/dashboard/components/navbar_component.py

```diff
@@ -43,11 +47,11 @@ def is_registry_item_visible(item: RegistryItem) -> bool:
 
 def generate_navbar_items(
     page_registry: PageRegistry, item_to_highlight: Optional[str] = None
-) -> list[dcc.Link]:
+) -> list[Link]:
```

Comment:
Inconsistent use of `dcc.Link` vs `Link`. Use one throughout the whole module.

Reply:
The autoformatter changed it